subworkflows/local/call_variants_gatk/tests/nextflow.config

```
@@ -0,0 +1,40 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
+----------------------------------------------------------------------------------------
+*/
+
+//
+// CALL_VARIANTS_GATK
+//
+
+process {
+    withName: '.*CALL_VARIANTS_GATK:GATK4_SPLITNCIGARREADS' {
+        ext.prefix = { "${meta.id}_splitncigar" }
+    }
+
+    withName: '.*CALL_VARIANTS_GATK:GATK4_HAPLOTYPECALLER' {
+        ext.prefix = { "${meta.id}_gatk" }
+        ext.args = { [
+            '--dont-use-soft-clipped-bases',
+            '--create-output-variant-index',
+            '--standard-min-confidence-threshold-for-calling 20'
+        ].join(' ').trim() }
+    }
+
+    withName: '.*CALL_VARIANTS_GATK:GATK4_VARIANTFILTRATION' {
+        ext.args = { [
+            '--window 3',
+            '--cluster 35',
+            '--filter-name "QD" --filter-expression "QD < 2.0"',
+            '--filter-name "FS" --filter-expression "FS > 30.0"'
+        ].join(' ').trim() }
+    }
```

Let's give these lowest priority so that they are not mistakenly used for other things than the tests, not sure if that could happen but 🤷 

```suggestion
    withName: 'GATK4_SPLITNCIGARREADS' {
        ext.prefix = { "${meta.id}_splitncigar" }
    }

    withName: 'GATK4_HAPLOTYPECALLER' {
        ext.prefix = { "${meta.id}_gatk" }
        ext.args = { [
            '--dont-use-soft-clipped-bases',
            '--create-output-variant-index',
            '--standard-min-confidence-threshold-for-calling 20'
        ].join(' ').trim() }
    }

    withName: 'GATK4_VARIANTFILTRATION' {
        ext.args = { [
            '--window 3',
            '--cluster 35',
            '--filter-name "QD" --filter-expression "QD < 2.0"',
            '--filter-name "FS" --filter-expression "FS > 30.0"'
        ].join(' ').trim() }
    }
```

---

It really shouldn't happen because this config is only called within the nf-test, but I can do it as it won't hurt 😄

---

I'm being paranoid 😄